gui/default/index.html

```diff
@@ -682,6 +682,14 @@ <h4 class="panel-title">
                       <th><span class="far fa-fw fa-clock"></span>&nbsp;<span translate>Uptime</span></th>
                       <td class="text-right">{{system.uptime | duration:"m"}}</td>
                     </tr>
+                    <tr>
+                      <th><span class="fas fa-fw fa-qrcode"></span>&nbsp;<span translate>Identification</span></th>
```

Comment:
Let's keep calling it "Device ID" to keep consistency with e.g. the device edit dialog.

Reply:
Isn't "Device ID" a little superfluous (being located under my device and other devices? "Identification" in the long form is actually used right there in the linked pop-up. Just "ID" is also an option though.

Reply:
Yeah I am mostly worried about translations. For English the link between "ID" and "Identification" is indeed obvious, but I wouldn't be surprised if "Device ID" and "Identification" get translated to terms that aren't obviously matched. Thus I'd prefer a bit of redundancy to ensure consistency.

Reply:
This will vary from language to language, but I can confirm that, e.g. in Korean, "ID" is translated as "ID" (i.e. is left untranslated), but "identification" is translated to a Korean word. There is no direct connection between the two, other than the placement in the Web GUI.

Reply:
My thinking was also that "Device" is just repeating the context, and "ID" is just an abbreviation of "Identifier" or "Identification".  Seeing that the modal that opens on click already had "Device Identification" as its title, I adapted to that *also for consistency*.

Changed to "Device ID".

Reply:
I agree with that reasoning (I wasn't aware "Device Identification" is already in use/translated). Feel free to revert that change, I'll re-approve. Or merge as is.

Reply:
Reverted, thanks.